### ISSUE TITLE
utils.uncomment not removing "line" comments

### ISSUE THREAD

**Issue by mathiasrw** [agershun/alasql#1796]
[alasql.utils.uncomment](https://github.com/AlaSQL/alasql/blob/develop/src/16comments.js#L17) is meant to remove `-- something` comments, but is not.

Expected to work:
```
res = alasql.utils.uncomment('a -- wow\nb');
// res == 'a \nb'
```
Please update [test 221](https://github.com/AlaSQL/alasql/blob/develop/test/test221.js) to show its working by removing the .skip in the first test. 

---

_Spawn from the work with https://github.com/AlaSQL/alasql/issues/1795_

**Comment by vishal6557** [comment]
Can I please work on this?

**Comment by mathiasrw** [comment]
Yes, please. 

I propose you also add a few more tests for the functionality so we know if we are breaking things while fixing.

**Comment by mathiasrw** [comment]
Fix released as part of v4.2.3
